### PR TITLE
remove duplicated indexes according to pghero

### PR DESCRIPTION
**Pull request by lfuelling**
When opening PgHero it warns about seven duplicate indexes and gives instructions how to migrate them away:
```
These indexes exist, but aren’t needed. Remove them with a migration for faster writes.
    rails generate migration remove_unneeded_indexes
And paste
[...]
```

The suggested name was already taken so I changed it to `remove_duplicated_indexes_pghero`, pasted the code generated by PgHero and ran `db:migrate` to update `schema.rb`.

### CODE REVIEW

**Comment by ProgVal** [comment]
ping.

People sometimes see this warning in pgHero and apply the migration themselves, which makes their schema diverge from what further migrations expect, which causes issues.

It would be nice to fix this warning (and make sure it doesn't reappear in futures releases) so it stops happening.

**Review by ClearlyClaire — APPROVED** [review]
I believe the reason why those duplicate indexes have been kept for so long is that a script was once used that declared any index not being an *exact* match on the foreign key was questionable.

However, multi-column indexes starting with the foreign key should be just as good, so removing those indexes should be fine.

**Comment by ThibG** [comment]
@lfuelling can you rebase this PR on master? The CI failure is something we fixed in master a while ago, it doesn't have to do with your code.

File: db/schema.rb [hunk 1]
```diff
@@ -10,8 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_05_10_181721) do
-
+ActiveRecord::Schema.define(version: 2020_06_05_155027) do
```
Comment (lfuelling) [inline db/schema.rb]: @ThibG According to the docs, the `version`should point to the latest migration, so I simply chose the later date. I hope that's how it's done 🙈

Comment (ThibG) [reply db/schema.rb]: That is fine, it doesn't matter much, as those migrations can be applied in any order.

**Review by Gargron — COMMENTED** [review]
The schema version in schema.rb didn't update so the migration might not get picked up?

**Comment by nightpool** [comment]
you should run db:rollback and db:migrate after editing any migration. (if
the migration in question isn't the most recent migration, you should use
db:migrate:down and specify a version instead)

**Pull request merged**